### DISCUSSION TITLE
Maximum bunde size of all classes (version 3.2.4)

### DISCUSSION THREAD

**Discussion by localusercamp** [tailwindlabs/tailwindcss#10465] [General]
Hello.

I tried to find information about how much KB **ALL** Tailwind CSS classes weight, but didn't find it anywhere. By **ALL** i mean all documented classes (no arbitary values). 
1.9.0 version of tailwind has such [page](https://v1.tailwindcss.com/docs/controlling-file-size) in docs. Is it size of javascript code that generates all classes, or maximum size of a `.css` file with all classes? Anyway i want to know the size of 3.2.4 version.
[CDN 3.2.4](https://cdn.tailwindcss.com/3.2.4) version ships with javascript code which generates classes and it's about 327.08 KB plain text size. But i'm curious what the size of a generated `.css` file with all TailwindCSS classes?

**Comment by cannap** [comment]
create a safeList with a regex which allows ervrything no clue this will work (.+?)

**Reply by localusercamp** [reply]
Genius idea! =D

**Comment by localusercamp** [comment]
If anybody interested this config produces 7 MB of CSS
```
safelist: [{
    pattern: /.*/,
}]
```
It includes all possible variations for all colors, modifiers, variants and so on
